### PR TITLE
Spring Data Cosmos - fix SpEL test error

### PR DESCRIPTION
**Pull request by saragluna**
There was this error when the spring-data-cosmos integration test run:
```
org.springframework.beans.factory.BeanExpressionException: Expression parsing failed; nested exception is org.springframework.expression.spel.SpelEvaluationException: EL1058E: A problem occurred when trying to resolve bean 'dynamicContainer':'Could not resolve bean reference against BeanFactory'
	at com.azure.spring.data.cosmos.repository.integration.SpELCosmosAnnotationIT.testDynamicContainerNameWithBeanExpression(SpELCosmosAnnotationIT.java:83)
Caused by: org.springframework.expression.spel.SpelEvaluationException: EL1058E: A problem occurred when trying to resolve bean 'dynamicContainer':'Could not resolve bean reference against BeanFactory'
	at com.azure.spring.data.cosmos.repository.integration.SpELCosmosAnnotationIT.testDynamicContainerNameWithBeanExpression(SpELCosmosAnnotationIT.java:83)
Caused by: org.springframework.expression.AccessException: Could not resolve bean reference against BeanFactory
	at com.azure.spring.data.cosmos.repository.integration.SpELCosmosAnnotationIT.testDynamicContainerNameWithBeanExpression(SpELCosmosAnnotationIT.java:83)
Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No bean named 'dynamicContainer' available
	at com.azure.spring.data.cosmos.repository.integration.SpELCosmosAnnotationIT.testDynamicContainerNameWithBeanExpression(SpELCosmosAnnotationIT.java:83)
```
It's difficult to reproduce this error locally and this PR is trying to fix this error.

--- 
I took @kushagraThapar's advice to run all tests in IDEA, and the issue could be reproduced.

### CODE REVIEW

**Review by chenrujun — APPROVED** [review]
LGTM.

File: sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/config/CosmosConfigurationSupport.java [hunk 1]
```diff
@@ -15,16 +19,15 @@
 import org.springframework.util.ClassUtils;
 import org.springframework.util.StringUtils;
 
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.Set;
+import java.util.*;
```
Comment (kushagraThapar) [inline sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/config/CosmosConfigurationSupport.java:22]: Please change it to separate imports, we do not use * imports.

File: sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/config/CosmosConfigurationSupport.java [hunk 2]
```diff
@@ -3,9 +3,13 @@
 
 package com.azure.spring.data.cosmos.config;
 
+import com.azure.spring.data.cosmos.CosmosFactory;
```
Comment (kushagraThapar) [inline sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/config/CosmosConfigurationSupport.java:6]: Probably an unused import, pls check.

**Comment by saragluna** [comment]
The `@ContextConfiguration` annotation we used will cache the context for later use, and if another context initialization happens before the SpEL test, it will change the context reference held by `ExpressionResolver`. And since the SpEL test will use the cached context, there's no chance to change `ExpressionResolver` again.

File: sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/config/CosmosConfigurationSupport.java (hunk 2 above)
Comment (saragluna) [reply sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/config/CosmosConfigurationSupport.java]: deleted.

File: sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/config/CosmosConfigurationSupport.java (hunk 1 above)
Comment (saragluna) [reply sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/config/CosmosConfigurationSupport.java]: solved.

**Pull request merged**